### ISSUE TITLE
Custom metrics are gone?

### ISSUE THREAD

**Issue by nuzolx** [microsoft/ApplicationInsights-dotnet#2061]
Hi,

Today I have updated my NuGet packages.

From:
```
    <PackageReference Include="Microsoft.ApplicationInsights.DependencyCollector" Version="2.14.0" />
    <PackageReference Include="Microsoft.ApplicationInsights.EventCounterCollector" Version="2.14.0" />
    <PackageReference Include="Microsoft.ApplicationInsights.PerfCounterCollector" Version="2.14.0" />
    <PackageReference Include="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel" Version="2.14.0" />
    <PackageReference Include="Microsoft.ApplicationInsights.WorkerService" Version="2.14.0" />
    <PackageReference Include="Microsoft.Extensions.Logging.ApplicationInsights" Version="2.14.0" />
```

To: 
```
    <PackageReference Include="Microsoft.ApplicationInsights.DependencyCollector" Version="2.15.0" />
    <PackageReference Include="Microsoft.ApplicationInsights.EventCounterCollector" Version="2.15.0" />
    <PackageReference Include="Microsoft.ApplicationInsights.PerfCounterCollector" Version="2.15.0" />
    <PackageReference Include="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel" Version="2.15.0" />
    <PackageReference Include="Microsoft.ApplicationInsights.WorkerService" Version="2.15.0" />
    <PackageReference Include="Microsoft.Extensions.Logging.ApplicationInsights" Version="2.15.0" />
```

And some other (from 3.1.7), but I think this is not relevant:
```
<PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="3.1.8" />
<PackageReference Include="Microsoft.Extensions.Hosting" Version="3.1.8" />
 <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="3.1.8" />
```

* Runtime version:  netcoreapp3.1.8
* Hosting environment : Windows Server

No code change!

Before this update, I have some customMetrics. Only default counters.

![image](https://user-images.githubusercontent.com/1068094/93620964-201ac200-f9db-11ea-996c-6b629543af62.png)

Since this update, I have only 'HeartbeatSate' and this new. Never seen it before.

![image](https://user-images.githubusercontent.com/1068094/93621082-5c4e2280-f9db-11ea-84bd-27c90a326f20.png)

And when I start my app:

![image](https://user-images.githubusercontent.com/1068094/93621343-b0f19d80-f9db-11ea-80ab-7beb043cec35.png)

No specific code just:
```
builder.Services.AddApplicationInsightsTelemetryWorkerService(instrumentationKey);
builder.AddFilter<ApplicationInsightsLoggerProvider>("", LogLevel.Information);
```

BR,

**Comment by cijothomas** [comment]
Starting with 2.15, there is no default set of EventCounters collected by default. 
https://docs.microsoft.com/en-us/azure/azure-monitor/app/eventcounters#default-counters-collected

Please explicitly add the counters you are interested in to get them back.

(As you probably realized already, EventCounters are custom metrics, (not StandardMetrics in application insights), hence they are billed. So we decided its best for customers to enable those counters they are interested in.

(Note: Standard Metrics are free, If ApplicationInsights ever promotoes some eventcounters into StandardMetric list, then we'll change the default behavior.)

**Comment by cijothomas** [comment]
HearbeatState was always on by default for a long time. Due to some bugs (which got fixed in 2.15), it was broken in some .NET Core/Asp.Net Core scenarios. But its working now. You can disable them if you want.

**Comment by nuzolx** [comment]
@cijothomas Thx you for the quick response.
I have open the doc before posting here, just for check that "the following counters are collected automatically by the SDK. ". This is still present. I didnt scroll...or I have read the french version "https://docs.microsoft.com/fr-fr/azure/azure-monitor/app/eventcounters" (outdated),. I don't remember.
